src/ViewModel/Converter/ArticleDownloadLinksListConverter.php

```diff
@@ -58,6 +58,10 @@ public function convert($object, string $viewModel = null, array $context = [])
                     $this->downloadLinkUriGenerator->generate(DownloadLink::fromUri($context['dar-download'])),
                     false,
                     ['article-identifier' => $object->getDoi(), 'download-type' => 'dar-download']
+                ),
+                new ViewModel\Link(
+                    'Learn more about DAR',
+                    $this->urlGenerator->generate('labs-post', ['id' => '7dbeb390'])
```

Comment:
shouldn't this always be an absolute URL on `elifesciences.org`? It doesn't exist on other environments (hence it could break end2end tests that check no 404s are there, etc)

Reply:
Does on preview though, and would take you to a different site otherwise. Done this elsewhere, but 🤷‍♂.

Reply:
Links are not checked for 404s so it should pass the `end2end` tests: https://github.com/elifesciences/elife-spectrum/blob/2ee92e3076b97477a0c226f04ef6c5f618204184/spectrum/checks.py#L845